oneanddone/tasks/forms.py

```diff
@@ -25,9 +25,11 @@ def __init__(self, *args, **kwargs):
 
 
 class FeedbackForm(forms.ModelForm):
+    completion_time_in_minutes = forms.IntegerField(
```

Comment:
I notice that if I enter a value that is too large, e.g., `999999999999999`, then an exception is thrown. I might have thought that the `max_length=5` in the model would have caused there to be an automatic validation for this, but that doesn't seem to be happening. Can you please see what can be done so that an exception will not be thrown for an invalid number?